CHANGELOG-1.1.md

```diff
@@ -0,0 +1,122 @@
+- Support initializing TiDB Cluster with TidbInitializer Custom Resource. ([#1403](https://github.com/pingcap/tidb-operator/pull/1403), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Refine the configuration schema of PD/TiKV/TiDB. ([#1411](https://github.com/pingcap/tidb-operator/pull/1411), [@aylei](https://github.com/aylei))
+- The instance label key of resources owned by `tidbcluster` is default to the cluster name now. ([#1419](https://github.com/pingcap/tidb-operator/pull/1419), [@aylei](https://github.com/aylei))
+- Custom resource `TidbCluster` is extended to support managing Pump cluster. ([#1269](https://github.com/pingcap/tidb-operator/pull/1269), [@aylei](https://github.com/aylei))
+- Fix the default tikv-importer configuration ([#1415](https://github.com/pingcap/tidb-operator/pull/1415), [@aylei](https://github.com/aylei))
+- expose ephemeral-storage in resource config ([#1398](https://github.com/pingcap/tidb-operator/pull/1398), [@aylei](https://github.com/aylei))
+- e2e case of operating tidb-cluster without helm ([#1396](https://github.com/pingcap/tidb-operator/pull/1396), [@aylei](https://github.com/aylei))
+- Expose terraform aliyun ACK version and specify the default version to '1.14.8-aliyun.1'. ([#1284](https://github.com/pingcap/tidb-operator/pull/1284), [@shonge](https://github.com/shonge))
+- scheduler: refine scheduler error message ([#1373](https://github.com/pingcap/tidb-operator/pull/1373), [@weekface](https://github.com/weekface))
+- Use cluster-role system:kube-scheduler to binding service account tidb-scheduler. ([#1355](https://github.com/pingcap/tidb-operator/pull/1355), [@shonge](https://github.com/shonge))
+- Add new CRD TidbInitializer. ([#1391](https://github.com/pingcap/tidb-operator/pull/1391), [@aylei](https://github.com/aylei))
+- Upgrade the default backup image to pingcap/tidb-cloud-backup:20191217 and facilitate -r option ([#1360](https://github.com/pingcap/tidb-operator/pull/1360), [@aylei](https://github.com/aylei))
+- Terraform: fix docker ulimit configuring for latest EKS AMI. ([#1349](https://github.com/pingcap/tidb-operator/pull/1349), [@aylei](https://github.com/aylei))
+- Support sync pump status to tidb-cluster. ([#1292](https://github.com/pingcap/tidb-operator/pull/1292), [@shonge](https://github.com/shonge))
+- `tidb-controller-manager` will automatically create and reconcile the tidb-discovery-service now ([#1322](https://github.com/pingcap/tidb-operator/pull/1322), [@aylei](https://github.com/aylei))
+- make backup and restore more universal and secure ([#1276](https://github.com/pingcap/tidb-operator/pull/1276), [@onlymellb](https://github.com/onlymellb))
+- pd and tikv configs are now managed in TidbCluster resource. ([#1330](https://github.com/pingcap/tidb-operator/pull/1330), [@aylei](https://github.com/aylei))
```

Comment:
```suggestion
- Manage PD and TiKV configuations in TidbCluster resource ([#1330](https://github.com/pingcap/tidb-operator/pull/1330), [@aylei](https://github.com/aylei))
```

Reply:
Please start a PR description with a verb and remove periods to keep a consistent style.